src/ts/AppBar/index.tsx

```diff
@@ -73,4 +79,6 @@ export default class AppBarOwn extends Component<any, any> {
   }
 }
 
-
+AppBarVG.childContextTypes = {
+  muiTheme: React.PropTypes.object.isRequired,
+};
```

Comment:
Do we need this? As far as we don't use theme in component we don't have to get it from context. Another thing is that we can wrap component using `muiThemeable()(Component)` to get theme from props (as described [here](http://www.material-ui.com/#/customization/themes))